torch/_inductor/kernel/quack/__init__.py

```diff
@@ -0,0 +1,3 @@
+__version__ = "0.1.7"
```

Comment:
Please also attach the license file from quack. (Otherwise I believe you're breaking the license).